ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml

```diff
@@ -5263,6 +5236,18 @@ periodics:
         value: ovn
       - name: CONFIG_SPEC
```

Comment:
I said to change this back to `CONFIG_SPEC`, but I missed the fact that this is now using a step registry based workflow. So it should in fact be `UNRESOLVED_CONFIG` in this case.